### PR TITLE
Feature/movie detail display

### PR DESCRIPTION
**Pull request by kendallha**
# Describe Changes

Please include changes made or which issues were fixed.

# Type of Change
- [x] Feature HTML
- [x] Feature JavaScript
- [x] Feature CSS
- [x] Feature React
- [ ] Refactor/Fix Bugs

# Please describe the change made

Please give a short description of the changes that were made below
- Feature:
 - Added new React component MovieDetails to use in conditional rendering
 - Set conditional rendering in App component render MovieDetails component when a movie thumbnail is clicked
 - Added click handling for back button on MovieDetails page to render MovieList component
 
 - Added styling for the MovieDetails component:
Screen Shot 2021-05-14 at 6 01 46 PM

# How has this been tested?

- [ ] In Browser
- [x] In Console

### CODE REVIEW

File: src/MovieDetail/MovieDetail.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
+.movie-detail-wrapper {
+  /* background-color: #324653; */
```
Comment (MarcelineBall) [inline src/MovieDetail/MovieDetail.css:2]: Might want to get rid of the comment out code

File: src/MovieDetail/MovieDetail.css [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+}
+
+.double-poster-wrapper {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.poster-wrapper {
+  /* height: 20%; */
```
Comment (MarcelineBall) [inline src/MovieDetail/MovieDetail.css:43]: We might want to get rid of the comment out code

File: src/MovieDetail/MovieDetail.css [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  justify-content: space-around;
+}
+
+.main-details {
+ font-style: italic;
+}
+
+/* .movie-details-aside {
+  padding-top: 4%;
+} */
```
Comment (MarcelineBall) [inline src/MovieDetail/MovieDetail.css:88]: We might want to get rid of the comment out code

File: src/MovieDetail/MovieDetail.css [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+  background-attachment: fixed;
+  background-size: cover;
+}
+
+.fas {
+  color: white;
+  margin-left: 5%;
+  margin-top: 3%;
+  width: 500px;
+}
```
Comment (MarcelineBall) [inline src/MovieDetail/MovieDetail.css:17]: We might want to add a hover state for the cursor to indicate that the arrow is a button

File: src/MovieDetail/MovieDetail.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import './MovieDetail.css';
+
+const MovieDetail = ( {movie, goBack} ) => {
+  return (
+    <div className='movie-detail-wrapper' style={{
+        backgroundImage: 'url('+ movie.backdrop_path +')'
+      }}>
+      <article className='movie-expanded'>
+      <i onClick={() => goBack()} className="fas fa-arrow-left fa-2x"></i>
```
Comment (MarcelineBall) [inline src/MovieDetail/MovieDetail.js:10]: We might want to add an ARIA role to indicate that the icon is a button. This icon may also give us some issues if the background being displayed is a light color

**Review by MarcelineBall — COMMENTED** [review]
The logic for this iteration looks good! There are a few changes in the HTML and CSS that could be made to improve UI and accessibility, but the overall design looks really nice!

File: src/MovieDetail/MovieDetail.css (hunk 4 above)
Comment (kendallha) [reply src/MovieDetail/MovieDetail.css:17]: Good idea! I added this to the poster images in the Movie component as well to indicate that the posters are clickable to get more info on the movie.

**Review by MarcelineBall — APPROVED** [review]
This looks good! Good call on adding the pointer indicator for the cursor on the movies

**Pull request merged**